mongo/integration/unified_spec_test.go

```diff
@@ -636,6 +621,7 @@ func lastTwoIDs(mt *mtest.T) (bson.RawValue, bson.RawValue) {
 	return first, second
 }
 
+// ! this entire function probably goes away
```

Comment:
I think you probably left this in by accident. I assume the legacy transactions tests still use this?